Sources/Mantis/CropView/CropView.swift

```diff
@@ -578,7 +579,7 @@ extension CropView {
         guard let croppedImage = image.crop(by: cropInfo) else {
             return (nil, transformation, cropInfo)
         }
-        
+                
```

Comment:
nit: xcode is screwing with your spaces

There is an option to disable this though: https://stackoverflow.com/questions/1390329/trim-trailing-spaces-in-xcode

Reply:
Thanks for the sugggestion!
I will try it.